internal/cmd/scan/scan.go

```diff
@@ -79,7 +81,7 @@ If the given path contains a git repository all flags but "integration" will be
 		`name of integration used to trigger scan. For example "GitHub Actions"`,
 	)
 	cmd.Flags().StringVarP(&jsonFilePath, JsonFilePathFlag, "j", "", "write upload result as json to provided path")
-
+	cmd.Flags().StringVar(&configFilePath, ConfigFilePathFlag, "debricked-config.yaml", "read debricked configuration from provided path")
```

Comment:
It is not supposed to work like this. The CLI is supposed to search for the file and automatically pick it up if it exists.